website_membership_registration/i18n/nl.po

```diff
@@ -137,31 +137,31 @@ msgstr ""
 #. module: website_membership_registration
 #: model_terms:ir.ui.view,arch_db:website_membership_registration.membership_registration_form
 msgid "<span>Phone</span>"
-msgstr ""
+msgstr "<span>Telefoon</span>"
 
 #. module: website_membership_registration
 #: model_terms:ir.ui.view,arch_db:website_membership_registration.membership_registration_form
 msgid "<span>Publish me as a member</span>"
-msgstr "<span>Publish me as a member</span>"
+msgstr "<span>Publiceer mij als lid</span>"
 
 #. module: website_membership_registration
 #: model_terms:ir.ui.view,arch_db:website_membership_registration.portal_my_details
 msgid "<span>Publish me</span>"
-msgstr "<span>Publiceer mij als lid</span>"
+msgstr "<span>Publiceer mij</span>"
 
 #. module: website_membership_registration
 #: model_terms:ir.ui.view,arch_db:website_membership_registration.membership_registration_form
 msgid ""
 "<span>State / Province</span>\n"
 "                                                    <span class=\"s_website_form_mark\"> *</span>"
 msgstr ""
-"<span>State / Provincie</span>\n"
+"<span>Provincie</span>\n"
```

Comment:
Staat / Provincie

Even though it is Dutch, if the customer is from the US, they would still have a state, I think.